cobalt/tools/on_device_tests_gateway_client.py

```diff
@@ -123,12 +123,12 @@ def _process_test_requests(args):
 
   for gtest_target in args.targets.split(','):
     _, target_name = gtest_target.split(':')
-    print(f'  Processing gtest_target: {gtest_target}')
+    # print(f'  Processing gtest_target: {gtest_target}')
```

Comment:
Remove (and the other commented out statement below)? I don't think we need this arg as the gtest filter will be displayed in the target log.